mw-webapp/src/component/table/renderCellValue/helpers/updateCell.ts

```diff
@@ -0,0 +1,41 @@
+import {updateCurrentProblem} from "src/component/table/renderCellValue/helpers/updateCurrentProblem";
+import {updateDayReport} from "src/component/table/renderCellValue/helpers/updateDayReport";
+import {updateJobDone} from "src/component/table/renderCellValue/helpers/updateJobDone";
+import {updateMentorComment} from "src/component/table/renderCellValue/helpers/updateMentorComment";
+import {updatePlanForNextPeriod} from "src/component/table/renderCellValue/helpers/updatePlanForNextPeriod";
+import {ColumnNameProps} from "src/component/table/renderCellValue/renderCellItem";
+import {CurrentProblem} from "src/model/businessModel/CurrentProblem";
+import {JobDone} from "src/model/businessModel/JobDone";
+import {MentorComment} from "src/model/businessModel/MentorComment";
+import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
+
+/**
+ * Update cell value
+ * @param {string} text
+ * @param {function} callback
+ * @param {JobDone | PlanForNextPeriod | CurrentProblem | MentorComment | string} arrayItem
+ * @param {string} parentUuid
+ * @param {keyof ColumnNameProps} columnName
+ * @param {number} index
+ */
+export const updateCell = (
```

Comment:
Not part of component

Reply:
Shouls I replace it to DAL layer or create a new folder (for view layer)?

Reply:
Maybe, I just mentioned, that currently it looks unappropriate